src/service/fixed-layer.js

```diff
@@ -135,6 +135,7 @@ export class FixedLayer {
       // Apply transform style to all fixed elements
       this.fixedElements_.forEach(e => {
         if (e.fixedNow && e.top) {
+          setStyle(e.element, 'transition', 'none');
```

Comment:
Shall we rename this method to something like `transitMutate`? Don't forget to update the doc.